### ISSUE TITLE
BrailleBook URL doesn't resolve

### ISSUE THREAD

**Issue by ChristophEwertowski** [hbz/lobid-resources#827]
`BrailleBook` is defined by http://purl.org/library/BrailleBook which doesn't resolve. I searched for "Braille" in Schema.org, bibframe, RDA and BibO which didn't lead to a result.

**Comment by ChristophEwertowski** [comment]
http://d-nb.info/gnd/4145976-3 is a property which resolves and also leads to "Blindenschrift". Additionally I renamed the label in the frontend from "Braille" to "Blindenschrift" as it is identical with the internal label and probably a bit easier to understand. See also https://github.com/hbz/lobid-resources/pull/884/files

**Comment by acka47** [comment]
As I just noticed this accidentally looking at the mail. We do need an `rdfs/owl:Class` and not a property or Concept/AuthorityResource.

**Comment by ChristophEwertowski** [comment]
Deployed to test, see `medium` at [TT001210514 (test, json)](http://test.lobid.org/resources/TT001210514.json). I used a lobid class which has to be created in the repository lobid vocabs.

**Comment by acka47** [comment]
> and not a property or Concept/AuthorityResource.

This is not correct what I said. We should make this a skos:Concept to be used with dct:medium.

**Comment by TobiasNx** [comment]
https://github.com/hbz/lobid-resources/blob/2ae7b86259115ee3fa8693e3ba16ebda2739ca54/src/main/resources/alma/fix/mediumAndType.fix#L134-L153

Braille is set as URI but the label is Blindenschrift. The MARC 007 info is broader: https://www.loc.gov/marc/bibliographic/bd007f.html

Do we need a broader concept for Bilndenschrift?
